app/services/websocket.py

```
@@ -22,13 +31,102 @@ async def connect(self, websocket: WebSocket, user_id: str):
+        if not self.is_market_streaming:
+            self.is_market_streaming = True
+            self._market_stream_task = asyncio.create_task(self._start_market_streaming())
 
     async def broadcast(self, message: dict, user_id: str):
         if user_id in self.active_connections:
+            broken_connections = []
             for connection in self.active_connections[user_id]:
-                await connection.send_json(message)
+                try:
+                    await connection.send_json(message)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)